### PR TITLE
feat: Prepare valid gnap request to access agent sdk request access method

### LINKED ISSUE #1709: Prepare valid gnap request to access agent sdk request access method
Figure out how to convert pubkey to jwk 

nonce: "foo-bar-baz", this has to be persisted

### PR DESCRIPTION
**Pull request by talwinder50**
closes #1709

Signed-off-by: talwinder50 <talwinderkaur50@gmail.com>
Screen Shot 2022-06-16 at 12 24 00 AM

I get 401 error which is expected until http signer is in ERROR access policy failed to handle access request: client request verification failure: request with body should have a content-digest header

### CODE REVIEW

File: test/fixtures/wallet-web/docker-compose-web.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ services:
       - DID_ANCHOR_ORIGIN=https://testnet.orb.local
       - EDGE_AGENT_SERVER=https://wallet-server.trustbloc.local:8090
       - HTTP_SERVER_HUB_AUTH_URL=https://auth.trustbloc.local:8044
+      - GNAP_ACCESS_TOKEN_TYPE=trustbloc.dev/types/gnap/wallet
```
[the remark this answers is not in the document]
Comment (talwinder50) [reply test/fixtures/wallet-web/docker-compose-web.yml]: Update

File: cmd/wallet-web/src/router/index.js [hunk 2]
```diff
@@ -23,11 +23,15 @@ router.beforeEach(async (to, from, next) => {
...
+    const gnapAccessTokenType = computed(() => store.getters['gnapAccessTokenType']);
+    const resp = await gnapRequestAccess(
+      signer,
+      gnapAuthServerURL.value,
+      gnapAccessTokenType.value
+    );
     // TODO Issue-1699 Save properties from resp to Vuex/local storage and call GNAPClient.continue() in separate func
   }
-  if (to.path === 'gnap/redirect') {
+  if (to.path === 'gnap/interact') {
```
[the remark this answers is not in the document]
Comment (talwinder50) [reply cmd/wallet-web/src/router/index.js]: This is when we need to call GNAPClient.continue(), I think whatever the auth server redirects to after wallet calls requestAccess()





[3:09](https://avast-software.slack.com/archives/D03CD8BJVAN/p1655320167990599)
It should be whatever the auth server redirects to after wallet calls requestAccess()

File: cmd/wallet-web/src/config/gnap-access-token.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (fqutishat) [inline cmd/wallet-web/src/config/gnap-access-token.json:1]: why config part of src

**Review by birtony — APPROVED** [review]

File: cmd/wallet-web/src/config/gnap-access-token.json (hunk 3 above)
Comment (talwinder50) [reply cmd/wallet-web/src/config/gnap-access-token.json:1]: This is as a backup if we dont find the config file in the configurable setup. 

Thats how other backup configs are in the wallet-web
Screen Shot 2022-06-16 at 3 21 24 PM

**Review by fqutishat — APPROVED** [review]

**Pull request merged**